src/test/java/com/github/onsdigital/JWTVerifierImplTests.java

```diff
@@ -0,0 +1,111 @@
+package com.github.onsdigital;
+
+import com.github.onsdigital.exceptions.JWTDecodeException;
+import com.github.onsdigital.exceptions.JWTTokenExpiredException;
+import com.github.onsdigital.exceptions.JWTVerificationException;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class JWTVerifierImplTests {
+    private final static String SIGNED_TOKEN = "eyJhbGciOiJSUzI1NiIsImtpZCI6IjEyMzRleGFtcGxlPSJ9.eyJzdWIiOiJhYWFhYWFhYS1iYmJiLWNjY2MtZGRkZC1lZWVlZWVlZWVlZWUiLCJkZXZpY2Vfa2V5IjoiYWFhYWFhYWEtYmJiYi1jY2NjLWRkZGQtZWVlZWVlZWVlZWVlIiwiY29nbml0bzpncm91cHMiOlsiYWRtaW4iLCJkYXRhIiwicHVibGlzaGluZyIsInRlc3QiXSwidG9rZW5fdXNlIjoiYWNjZXNzIiwic2NvcGUiOiJhd3MuY29nbml0by5zaWduaW4udXNlci5hZG1pbiIsImF1dGhfdGltZSI6MTU2MjE5MDUyNCwiaXNzIjoiaHR0cHM6Ly9jb2duaXRvLWlkcC51cy13ZXN0LTIuYW1hem9uYXdzLmNvbS91cy13ZXN0LTJfZXhhbXBsZSIsImV4cCI6Nzk1NzA3MTI5MSwiaWF0IjoxNTYyMTkwNTI0LCJvcmlnaW5fanRpIjoiYWFhYWFhYWEtYmJiYi1jY2NjLWRkZGQtZWVlZWVlZWVlZWVlIiwianRpIjoiYWFhYWFhYWEtYmJiYi1jY2NjLWRkZGQtZWVlZWVlZWVlZWVlIiwiY2xpZW50X2lkIjoiNTdjYmlzaGs0ajI0cGFiYzEyMzQ1Njc4OTAiLCJ1c2VybmFtZSI6ImphbmVkb2VAZXhhbXBsZS5jb20ifQ.MrdJLEQ_YIG9uxJYguy9343CsW3WQ_TbaVKvbk0-ie_y1_WZF8wbGDybxfBeAQ9cy63f-bsuw4xQU8AavDNolR7F7bwWLPDAwkywauQXnUseDuEiNaA7xq386I0WmrUsTpNmCT7NgHDkDiBLSItxfd1aUxi2Z3lxr49TnzxKFmwZOGNE7k7xSlqzlwOmk78DvvI5RFknmzl1B3LJy7cqEnLK19LqcJcZlC5fzZH23fc8F7wAoKH755ARCVm2HrU2r_8pBXKl89siD0D6oy2f9NWUQikdP9XuCIPnDXNDSzdaDmqxz82vpm2rhWAwxfXLoIyR1eu9Za8VcN1Mk6oWrA";
```

Comment:
Could we put some line breaks between each of these class vars please - ATM its a wall of text.